packages/d3fc-webgl/src/buffers/elementIndicesBuilder.js

```diff
@@ -0,0 +1,54 @@
+export default () => {
+    let buffer = null;
+    let data = null;
+    let changed = false;
+
+    const base = (gl, cornerCount, elementCount) => {
+        if (!changed) {
+            return;
+        }
+        if (buffer == null) {
+            buffer = gl.createBuffer();
+        }
+        gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, buffer);
+
+        const vertexCount = data.length;
```

Comment:
Is this what we'd normally call this `verticesPerElement`? Should we pass it in here and assert that data is the right length?

Reply:
Unfortunately no, as per my comment above.

Also, due to the *ANGLE work, this line has been removed.